recipe/meta.yaml

```diff
@@ -86,7 +89,16 @@ outputs:
         {% set tests_to_skip = "test_url_is_accessible" %}
         # osx warns with nnpack if there is no AVX2, see conda-forge/pytorch-cpu-feedstock#56
         {% set tests_to_skip = tests_to_skip + " or test_adjust_sharpness" %}  # [osx]
-        - pytest -k "not ({{ tests_to_skip }})" test/
+        # 2021/10/28 hmaarrfk: I am able to run it locally on a large machine.
+        # It seems to fail around testing of vgg
+        # This test seems to just destroy the memory of the system.
+        {% set tests_to_skip = tests_to_skip + " or test_forward_backward" %}
+        {% set tests_to_skip = tests_to_skip + " or test_jit_forward_backward" %}
+        # 2021/10/29 hmaarrfk
+        # CIs do not have enough resources to run the full suite of model tests
+        - rm -f test/test_models.py  # [unix]
+        - del /f test\test_models.py  # [win]
+        - pytest --verbose -k "not ({{ tests_to_skip }})" test/
```

Comment:
```suggestion
        - pip install -U av pytest requests scipy pytest-mock
        - pytest --verbose -k "not ({{ tests_to_skip }})" test/
```

Reply:
I think --- I _believe_ --- this will also fix the issues with the other runs. There is this statement atop all these tests: 

```
try:
    import av

    # Do a version test too
    io.video._check_av_available()
except ImportError:
    av = None

```

And we get some errors about some type "None" etc

Reply:
This essentially means one really important thing for consideration: if we rely on `av` in actual building, then we cannot merge this until we fix the issues in the av-feedstock --- whatever they are --- but if `av` is some sort of an additional backend, then we can merge. I am currently running a test with basically no `av` to see what happens.

Reply:
(also maybe delete these reqs from the list above to avoid potential conflicts)

Reply:
(oh and please add python and pip to the list too, lol). Here's exactly what I had in that test section:
```yaml
    test:
      requires:
        # - pytest
        # - av
        # - requests
        # - scipy
        # - pytest-mock
        - python
        - pip
      source_files:
        - test
      imports:
        - torchvision
        - torchvision.datasets
        - torchvision.models
        - torchvision.transforms
        - torchvision.utils
      commands:
        # skip test_url_is_accessible instead of hitting 20+ servers per run, since
        # each server might be occasionally unresponsive and end up failing our CI
        {% set tests_to_skip = "test_url_is_accessible" %}
        # osx warns with nnpack if there is no AVX2, see conda-forge/pytorch-cpu-feedstock#56
        {% set tests_to_skip = tests_to_skip + " or test_adjust_sharpness" %}  # [osx]
        # 2021/10/28 hmaarrfk: I am able to run it locally on a large machine.
        # It seems to fail around testing of vgg
        # This test seems to just destroy the memory of the system.
        {% set tests_to_skip = tests_to_skip + " or test_forward_backward" %}
        {% set tests_to_skip = tests_to_skip + " or test_jit_forward_backward" %}
        # 2021/10/29 hmaarrfk
        # CIs do not have enough resources to run the full suite of model tests
        - rm -f test/test_models.py  # [unix]
        - del /f test\test_models.py  # [win]
        - pip install -U av pytest requests scipy pytest-mock
        - pytest --verbose -k "not ({{ tests_to_skip }})" test/

```
)

Reply:
Also, dropping `av` altogether reverts back to the initial set of failures. This indicates there are two problems here: One upstream to do with the `try` statement and second, more pressing, is the conda-forge's av-feedstock.

Reply:
@hmaarrfk confirmed this works also on linux. We are ready to roll. 👍

Reply:
unfortunately, we can't depend on pip packages at conda forge. we should simply skip the tests and report to our av feedstock that we think the av package is broken.

please provide me the appropriate skip commands.

Reply:
> unfortunately, we can't depend on pip packages at conda forge. we should simply skip the tests and report to our av feedstock that we think the av package is broken.
> 
> please provide me the appropriate skip commands.

I understand --- however, this is not "depending on pip" per se. It is just for the testing portion. I do realize though that this likely can introduce problems for at least a fraction of end users if they download the av-feedstock. I think skipping the tests might be prudent option. Let me figure out how to get those :)

Reply:
It effectively means: 

> We know our conda packages are broken, so instead of trying to understand why, we are shipping broken packages to end users, and asking them to run in circles installing packages similar to what is happening with other packages. One package from here, the other from there, and eventually you have an environment that they can't reproduce.

Reply:
Yep, in the case of av, yes. However, I personally feel that'a an okay compromise when it comes to these annoying packages (pytorch, tensorflow, julia, etc.). That is, shipping packages that can work for 90% of the time is better than delaying indefinitely. However, we should try to fix this asap in coordination with other feedstocks and immediately ship out updates.